akka-http-core/src/main/scala/akka/http/scaladsl/model/ErrorInfo.scala

```diff
@@ -72,4 +72,8 @@ object EntityStreamException {
   def apply(summary: String, detail: String = ""): EntityStreamException = apply(ErrorInfo(summary, detail))
 }
 
+case class EntityStreamSizeException(limit: Long, actualSize: Option[Long] = None) extends RuntimeException {
+  override def toString = s"EntityStreamSizeException($limit, $actualSize)"
```

Comment:
Unfortunately yes. The compiler doesn't add a `toString` because `RuntimeException` already comes with a non-default implementation.

```scala
scala> case class Foo(i: Int) extends RuntimeException
defined class Foo

scala> Foo(42)
res0: Foo = Foo
```